bestbook.egg-info/requires.txt

```diff
@@ -0,0 +1,3 @@
+virtualenv>=1.11.6
+pep8>=1.5.7
```

Comment:
[Our continuous integration uses flake8](https://github.com/Open-Book-Genome-Project/TheBestBookOn.com/blob/master/.github/workflows/lint_python.yml#L12) which bundles both pyflakes and pycodestyle which replaces the deprecated pep8.  These tools are test-time requirements, not runtime requirements so while they should be installed on developer machines, they need not be installed on end user machines or on production servers.